code/game/objects/items/weapons/swords_axes_etc.dm

```diff
@@ -83,6 +89,20 @@
 		add_blood(blood_color)
 	return
 
+/obj/item/weapon/melee/telebaton/proc/stun(mob/living/target, mob/living/user)
+	if(ishuman(target))
+		var/mob/living/carbon/human/H = target
+		if(H.check_shields(src, 0, "[user]'s [name]"))
+			return FALSE
+	user.visible_message("<span class='danger'>[user] knocks down [target] with [src]!</span>",\
+							"<span class='danger'>You knock down [target] with [src]!</span>")
```

Comment:
```suggestion
	user.visible_message("<span class='danger'>[user] knocks down [target] with \the [src]!</span>",\
							"<span class='danger'>You knock down [target] with \the [src]!</span>")
```
Also you should just be able to use SPAN_WARNING or whatever here insteaf of this span class stuff